src/agent/DiskId.cpp

```diff
@@ -0,0 +1,14 @@
+
+#include "DiskId.hpp"
+
+DiskId::DiskId(const std::string& system_id)
```

Comment:
Lack of convention in argument naming. It would be grat to have CamelCase for example not python_style.